tensorboard/plugins/metrics/metrics_plugin.py

```diff
@@ -305,10 +306,16 @@ def is_active(self):
     def _serve_tags(self, request):
         ctx = plugin_util.context(request.environ)
         experiment = plugin_util.experiment_id(request.environ)
-        index = self._tags_impl(ctx, experiment=experiment)
+
+        run_ids = None
+        query_params = urllib.parse.parse_qs(request.environ["QUERY_STRING"])
```

Comment:
You shouldn't need to parse the query string manually, you can just do `request.args.get("runIds")`. I believe it returns None if the parameter isn't present, you might want to confirm that.

Reply:
This would be great but as far as I can tell it always returns either `string` or `None`. I would like to get a list of strings. I could JSON encode a list on the frontend, send that, then decode that on the backend, but that doesn't seem cleaner to me. Do you have any thoughts?